plugin/evm/vm_upgrade_bytes_test.go

```diff
@@ -93,6 +94,9 @@ func TestVMUpgradeBytesPrecompile(t *testing.T) {
 	defer func() {
 		metrics.Enabled = true
 	}()
+	defaultValidatorPrivateKeyFile = "validator_private_key"
```

Comment:
`defaultValidatorPrivateKeyFile` is now required now cuz `createValidatorPrivateKey` is not using the argument passed

Reply:
fixed